unittests/wasm_tests.cpp

```diff
@@ -1952,7 +1952,9 @@ BOOST_AUTO_TEST_CASE( billed_cpu_test ) try {
    BOOST_CHECK_LT( max_cpu_time_us + 1, cpu_limit ); // max_cpu_time_us+1 has to be less than cpu_limit to actually test max and not account
    // indicate explicit billing at max + 1
    BOOST_CHECK_EXCEPTION( push_trx( ptrx, fc::time_point::maximum(), max_cpu_time_us + 1, true, 0 ), tx_cpu_usage_exceeded,
-                          fc_exception_message_starts_with( "billed") );
+                          [](const tx_cpu_usage_exceeded& e){ fc_exception_message_starts_with starts("billed");
+                                                              fc_exception_message_contains contains("reached on chain max_transaction_cpu_usage");
+                                                              return starts(e) && contains(e); } );
```

Comment:
I probably would have written:

```
return fc_exception_message_starts_with("billed")(e) && 
       fc_exception_message_contains("reached on chain max_transaction_cpu_usage")(e);
```
but I guess it is a matter of preference.

Reply:
This is test code and the diff in perf I don't think is worth changing it for.

Reply:
Actually there is probably very little diff in perf , and I 100% agree it is a matter of preference.